src/Insides/FormatNegotiator.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Franzl\Middleware\Whoops;
+namespace Franzl\Middleware\Whoops\Insides;
```

Comment:
Would `Private` be a valid namespace here? (I am worrying it's not because it's a keyword.) If so, I would rather have that. :)